PCP/server/src/server.py

```diff
@@ -47,6 +49,9 @@
 api.add_resource(Prices, '/prices')
 api.add_resource(UserFavorites, '/user_favorites')
 api.add_resource(UserFavoritesById, '/user_favorites/<int:user_id>')
+api.add_resource(Cart, '/cart/<int:user_id>')
+api.add_resource(AddToCart, '/add-to-cart/<int:user_id>/<int:product_id>/<int:quantity>')
+api.add_resource(RemoveFromCart, '/remove-from-cart/<int:user_id>/<int:product_id>')
```

Comment:
please do not make a new api for remove it should be in a PUT request to delete it !! not in POST !!

Reply:
same for add too